app/Http/Controllers/Api/AssetsController.php

```diff
@@ -852,7 +847,28 @@ public function checkin(Request $request, $asset_id)
             return response()->json(Helper::formatStandardApiResponse('success', ['asset'=> e($asset->asset_tag)], trans('admin/hardware/message.checkin.success')));
         }
 
-        return response()->json(Helper::formatStandardApiResponse('success', ['asset'=> e($asset->asset_tag)], trans('admin/hardware/message.checkin.error')));
+        return response()->json(Helper::formatStandardApiResponse('error', ['asset'=> e($asset->asset_tag)], trans('admin/hardware/message.checkin.error')));
```

Comment:
This was incorrectly set to success in the source, when in reality it should be an error.